bin/build-docs-site.sh

```diff
@@ -21,16 +21,16 @@ DIR=`dirname $0`
 REPO_ROOT=$DIR/..
 pushd $REPO_ROOT >/dev/null
 
-# Update apidocs
-lerna bootstrap
-lerna run --scope @loopback/docs prepack
-
 # Clean up sandbox/loopback.io directory
 rm -rf sandbox/loopback.io/
 
 # Shadow clone the `strongloop/loopback.io` github repo
 git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
 
+# Update apidocs
+lerna bootstrap --scope @loopback/tsdocs
```

Comment:
This is problematic as `lerna bootstrap --scope @loopback/tsdocs` does not link all packages.